site/en/blog/new-in-chrome-91/index.md

```diff
@@ -0,0 +1,190 @@
+description: >
+  Chrome 91 is rolling out now! Web apps that interact with files, can now
+  suggest file names and directories when using the file system access API.
+  Plus, you can now read files from the clipboard. If your site has more than
+  one domain, and shares the same account management backend, you can tell
+  Chrome they're the same, allowing the password manager to suggest the right
+  credentials. All the videos from I/O are now available. And there's plenty
+  more.
+layout: 'layouts/blog-post.njk'
+date: 2021-05-25
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/hcxe0ZGQy0gh5pemX88c.png'
+alt: >
+  New in Chrome hero logo
+tags:
+  - new-in-chrome
+  - chrome-91
+---
+
+{% YouTube id='vy6FXa0n1r0' %}
+
+Here's what you need to know:
+
+* Web apps that interact with files, can now
+  [suggest file names and directories](#filenames) when using the file system access API.
+* You can now [read files from the clipboard](#clipboard).
+* If your [site has more than one domain](credentials), and shares the same
+  account management backend, you can tell Chrome they're the same, allowing
+  the password manager to suggest the right credentials.
+* All the videos from I/O are now available.
+* And there's plenty [more](#more).
+
+I'm [Pete LePage](https://twitter.com/petele), working, and shooting
+from home, let's dive in and see what's new for developers in Chrome 91!
+
+## Suggested names for File System Access API {: #filenames }
+
+One of my favorite APIs to come out of the Fugu project this year is the
+file system access APIs. Once the user has granted permission, apps can
+interact with files on the users local device, in the same way other installed
+apps do, allowing you to create a more natural user experience.
+
+Starting in Chrome 91, you can now suggest the name and location of a file or
+directory to interact with. You can do this by passing a `suggestedName`
+property as part of the `showSaveFilePicker` options.
+
+```js/1
+const fileHandle = await self.showSaveFilePicker({
+  suggestedName: 'Untitled Text.txt',
+  types: [{
+    description: 'Text documents',
+    accept: {
+      'text/plain': ['.txt'],
+    },
+  }],
+});
+```
+
+The same goes for the default starting directory. For example, a text editor
+probably wants to start the file save or file open dialog in the `documents`
+folder. Whereas an image editor probably wants to start in the `pictures`
+folder. You can suggest a default start directory by passing a `startIn`
+property.
+
+```js/1
+const fileHandle = await self.showOpenFilePicker({
+  startIn: 'documents'
+});
+```
+
+Check out Tom's [File System Access][fs-post] post for complete details.
+
+## Reading files from the clipboard {: clipboard }
+
+There's one other cool new API for interacting with files that lands in
+Chrome 91. On desktop, web apps can now read files from the clipboard. Reading
+files from the clipboard has been available in Safari since 2018.
```

Comment:
```suggestion
There's one other cool new API for interacting with files that lands in
Chrome 91. On desktop, web apps can now read files from the clipboard. (Reading
files from the clipboard has been available in Safari since 2018.)
```